octobot_trading/exchanges/util/exchange_util.py

```diff
@@ -294,6 +297,8 @@ async def get_historical_ohlcv(
             while candles and candles[-1][common_enums.PriceIndexes.IND_PRICE_TIME.value] * 1000 > end_time:
                 candles.pop(-1)
                 reached_max = True
+            if candles[-1][common_enums.PriceIndexes.IND_PRICE_TIME.value] >= max_theoretical_time:
```

Comment:
:open_mouth:

Reply:
bingx raises when requesting for ohlcv with empty result